lib/validate-unit-identifiers.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+/**
+ * Module exports
+ */
+
+module.exports = validateUnitIdentifiers;
+
+/**
+ * @param {Array} rules
+ */
+
+function validateUnitIdentifiers(rules) {
+  rules.forEach(function (rule) {
+    rule.declarations.forEach(function (declaration) {
+      if (declaration.type !== 'declaration') {
+        return;
+      }
+
+      var val = declaration.value;
+      var pos = declaration.position.start;
+      var property = declaration.property;
+
+      var openParen = false;
+      var group = '';
+      var groups = [];
+      val.split('').forEach(function(c) {
+        if (c === '(') {
+          openParen = true;
+        }
+        if (c === ')') {
+          openParen = false;
+        }
+        if (c === ' ' && !openParen) {
+          groups.push(group);
+          group = '';
+          return;
+        }
+        group += c;
+      });
+      groups.push(group);
```

Comment:
Maybe there is some clever regex here but this was easier to code out so :koala: